src/main/java/org/gradle/profiler/report/CsvGenerator.java

```diff
@@ -93,6 +93,7 @@ protected void write(BenchmarkResult benchmarkResult, BufferedWriter writer) thr
         statistic(writer, "75th percentile", statistics, v -> v.getPercentile(75));
         statistic(writer, "max", statistics, BuildScenarioResult.Statistics::getMax);
         statistic(writer, "stddev", statistics, BuildScenarioResult.Statistics::getStandardDeviation);
+        statistic(writer, "p-value (Mann Whitney U test)", statistics, BuildScenarioResult.Statistics::getPValue);
```

Comment:
Can we mention somehow that this represents the confidence?

I would probably only call it "confidence" (or "p-value") and document somewhere else what it means. Having this complicated description breaks up the layout of the CSV quite a bit.

Reply:
I am calling it `confidence` now.